src/components/cc-img/cc-img.js

```diff
@@ -20,7 +20,7 @@ export class CcImg extends LitElement {
     return {
       skeleton: { type: Boolean, reflect: true },
       src: { type: String },
-      text: { type: String },
+      accessibleName: { type: String },
```

Comment:
suggestion: maybe we could add an attribute name `attribute: 'accessible-name` as it's two words?
```suggestion
      accessibleName: { type: String, attribute: 'accessible-name' },
```